src/modules/sensors/voted_sensors_update.cpp

```diff
@@ -142,6 +142,11 @@ void VotedSensorsUpdate::parameters_update()
 
 	_board_rotation = board_rotation_offset * _board_rotation;
 
+	// initialze all mag rotations with the board rotation in case there is no calibration data available
+	for (int topic_instance = 0; topic_instance < MAG_COUNT_MAX; ++topic_instance) {
+		_mag_rotation[topic_instance] = _board_rotation;
```

Comment:
I think yes, because if there is no valid calibration you have the rotation set to 0, which leads to 'stale mag' error messages. It's something we need to avoid as it only confuses devels & users. Plus it makes you think something is wrong with a mag, but in reality there is problem at all.